pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -129,6 +136,27 @@ func TestPipeline_Validate_Failure(t *testing.T) {
 		p: &Pipeline{
 			ObjectMeta: metav1.ObjectMeta{Name: "pipeline"},
 		},
+	}, {
+		name: "pipelinetask without name",
+		p: &Pipeline{
+			Spec: PipelineSpec{
+				Tasks: []PipelineTask{{Name: "", TaskRef: &TaskRef{Name: "valid-task"}}},
```

Comment:
This test is already done in `TestValidatePipelineTasks_Failure` (line 320).  Perhaps your other two tests should go in that method too.

Reply:
Good catch! Fixed.